### PR TITLE
ptl-tool: Validate Reviewed-by

### PR DESCRIPTION
**Pull request by joscollin**
Add a simple validation for the Reviewed-by input.

Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

File: src/script/ptl-tool.py [hunk 1]
```diff
@@ -70,6 +70,8 @@
 import sys
 
 from os.path import expanduser
+from email.utils import parseaddr
+from email_validator import validate_email, EmailNotValidError
```
Comment (tchaikov) [inline src/script/ptl-tool.py]: i'd suggest avoid introducing this dependency.

Comment (joscollin) [reply src/script/ptl-tool.py]: @tchaikov I had checked out a few email validators and this seems to be the better one. Do you suggest any other ?

Comment (joscollin) [reply src/script/ptl-tool.py]: @tchaikov Another point is: This doesn't affect the users of Ceph, as the script is going to be used only by `wip-*-testing` people.

**Comment by joscollin** [comment]
@batrick ping. I know, you might be busy. But please review, when you have time. Thanks.

File: src/script/ptl-tool.py (hunk 1 above)
Comment (batrick) [reply src/script/ptl-tool.py]: I don't really want to keep adding obscure dependencies if we can avoid it. I think just relying on the PTL to self-validate `Reviewed-by`s is fine.

**Comment by joscollin** [comment]
I'm closing this then.

**Pull request closed without merging**